### PR TITLE
Updating FAQ

### PR DESCRIPTION
**Pull request by bitcoinshirt**
I've been following people questions about BTCPay for a few months now. While GitHub is probably not the best way in which we can provide good FAQ and documentation for people not familiar with GitHub, I wanted to leave it here so that others can build on it and improve it. 

I re-arranged the FAQ and added .bit more generic questions, followed by technical issues.

I'm leaving it here, and I plan to work on it a bit more with the help of others.

There are two sections in particular which need to be reviewed : 
1. "Network Cost"
2. "Minimal Requirments"

I am not confident I got those two right, so please take a closer look at them and suggest or do the changes.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -2,11 +2,151 @@
 
 See also [The Merchants Guide to accepting Bitcoin directly with no intermediates through BTCPay](https://www.reddit.com/r/Bitcoin/comments/81h1oy/the_merchants_guide_to_accepting_bitcoin_directly/).
 
+## What is BTCPay Server?
+
+BTCPay Server is a free and open-source cryptocurrency payment processor which allows you to receive payments in Bitcoin and altcoins directly, with no fees, transaction cost or a middleman.
```
Comment (NicolasDorier) [inline FAQ.md]: Maybe mention lightning network?

File: FAQ.md [hunk 2]
```diff
@@ -2,11 +2,151 @@
...
+
+Also, once the full node is synchronized to the network, you can downgrade the machine on Azure to take a less expensive one.
+
+Please check out [How to penny-pinch your Azure deployment](AzurePennyPinching.md).
+
+## What are the minimal requirements to run BTCPay?
+
+If you would like to run Bitcoin and Lightning Network nodes, the minimal requirements are :
+- 2GB Ram
+- 250 GB of storage (for the initial sync) can be pruned to 80GB.
```
Comment (NicolasDorier) [inline FAQ.md]: 80 GB of storage (with pruning enabled). Actually there is not lot of reason to not prune imho.

File: FAQ.md [hunk 3]
```diff
@@ -2,11 +2,151 @@
...
+- Docker
+
+## How to install BTCPay Server?
+
+There are several videos and article online which you can check out:
+
+- [How to install BTCPay on Azure - guide](https://github.com/btcpayserver/btcpayserver-doc/blob/master/AzureDeployment.md)
+- [How to install BTCPay on Azure - video](https://www.youtube.com/watch?v=Bxs95BdEMHY)
+- [How to install BTCPay on LunaNode VPS - article](https://medium.com/@BtcpayServer/hosting-btcpayserver-on-lunanode-bf9ef5fff75b)
```
Comment (NicolasDorier) [inline FAQ.md]: You should add your channel and this one https://www.youtube.com/channel/UCpG9WL6TJuoNfFVkaDMp9ug

File: FAQ.md [hunk 4]
```diff
@@ -2,11 +2,151 @@
...
+
+Re-using the address for receiving payments is a privacy issue. Providing a different address manually to each customer is not an optimal solution. Imagine having to send an e-mail that wants to pay you with cryptocurrency.
+
+BTCPay solves the address re-use and automates the checkout process. The software is a complete, automated invoicing system. It integrates into your checkout process and customers can pay you in Bitcoin or altcoins in few clicks.
+
+After the payment, the software notifies your store that the order has been paid/completed. Depending on the e-commerce software you're using, it can also change the order status. You worry about getting items shipped, leave the invoicing and payment processing to BTCPay.
+
+## Does BTCPay need my private key?
+
+Absolutely not. The fact that BTCPay Server never needs access to your master private key is a huge security advantage. Even if your server gets hacked, your funds from the on-chain transactions are always safe. Securing your on-chain funds, comes down to [securing your wallet](https://btcinformation.org/en/secure-your-wallet).
```
Comment (NicolasDorier) [inline FAQ.md]: Exception: If you have a lightning node, BTCPay technically have access to the keys of LN.

File: FAQ.md [hunk 5]
```diff
@@ -2,11 +2,151 @@
...
+- Uniform Fiscal Object (UFO)
+- Viacoin (VIA)
+
+## Does BTCPay Server support crypto to fiat conversion?
+
+At this time, it is not possible to instantly convert your payments into fiat through BTCPay. This feature is planned for the future, but there is no ETA.
+
+## Where can I get help and support?
+
+BTCPay is an open-source project. It is not a company; there is no e-mail, live-chat or phone support. The software relies on a network of contributors and users to provide support.
```
Comment (NicolasDorier) [inline FAQ.md]: here as well https://github.com/btcpayserver/btcpayserver-doc/blob/master/Support.md

**Comment by NicolasDorier** [comment]
Reviewed, Very cool! Some links to add

File: FAQ.md (hunk 2 above)
Comment (bitcoinshirt) [reply FAQ.md]: @NicolasDorier Should I leave Docker as a requirement?
- 2GB Ram
- 80 GB of storage (with pruning enabled)
- Docker

File: FAQ.md [hunk 6]
```diff
@@ -2,11 +2,174 @@
...
+- Bitcoin (BTC)
+- Bitcoin Gold (BTG)
+- Dash (DASH)
+- Dogecoin (DOGE)
+- Feathercoin (FTC)
+- Groestlcoin (GRS)
+- Litecoin (LTC)
+- Monacoin (MONA)
+- Polis (POLIS)
+- Uniform Fiscal Object (UFO)
```
Comment (NicolasDorier) [inline FAQ.md]: Need to remove this one actually (UFO)

**Pull request merged**